lib/vertex_client/fallback_response.rb

```diff
@@ -4,6 +4,9 @@ class FallbackResponse
 
     attr_reader :body, :total_tax, :total, :subtotal, :line_items
 
+    # Temporary hack for tax_area_lookup
+    attr_reader :tax_area_id
+
```

Comment:
Now that we have a new endpoint in the mix, a lot of our previous abstractions are now broken. We will need to follow up with a refactor to get this codebase into good shape.